packages/firestore/src/lite-api/vector_value.ts

```diff
@@ -19,7 +19,7 @@ import { isPrimitiveArrayEqual } from '../util/array';
 
 /**
  * Represents a vector type in Firestore documents.
- * Create an instance with {@link FieldValue.vector}.
+ * Create an instance with `VectorValue.fromArray(array)`.
```

Comment:
I think the correct link should be `{@link vector}`? I think when we moved from compat to modular, we took a lot of symbols that used to be on the `FieldValue` namespace (serverTimestamp, arrayUnion, etc) and moved them to top level exports and this fits the pattern. @MarkDuckworth - confirm?

Reply:
This is correct. Good catch @hsubox76. May be copy-pasta or just a mix up when moving between Firestore SDK.